sig-security/security-audit-2021/RFP.md

```diff
@@ -75,12 +78,12 @@ Proposals should be submitted to kubernetes-security-audit-2021@googlegroups.com
 
 * 2021/02/08: RFP Open, Question period open
 * 2021/03/26: Question period closes
```

Comment:
Perhaps we want to extend question period while we're at it? Don't think we discussed that, but just occurred as I read the diff...

Reply:
I'll update this PR to have the Question period close to be the same as the RFP Close date